### ISSUE TITLE
After upgrading  flutter 2.5.1

### ISSUE THREAD

**Issue by glin94** [EMALLStudio/appmetrica_sdk#13]
```
[✓] Flutter (Channel stable, 2.5.1, on macOS 11.6 20G165 darwin-x64, locale ru-RU)
    • Flutter version 2.5.1 at /Users/linar/development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ffb2ecea52 (12 дней назад), 2021-09-17 15:26:33 -0400
    • Engine revision b3af521a05
    • Dart version 2.14.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/linar/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    • CocoaPods version 1.10.1
```

```
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **
Xcode's output:
↳
    /Users/linar/.pub-cache/hosted/pub.dartlang.org/appmetrica_sdk-0.0.7/ios/Classes/AppmetricaSdkPlugin.m:64:15: warning: unused variable 'maxReportsInDatabaseCount' [-Wunused-variable]
        NSInteger maxReportsInDatabaseCount = [call.arguments[@"maxReportsInDatabaseCount"] integerValue];
                  ^
2
    1 warning generated.
    ld: building for iOS Simulator, but linking in dylib built for iOS, file '/Users/linar/Projects/lariba/ios/Pods/YandexMobileMetrica/ios/dynamic/YandexMobileMetrica.framework/YandexMobileMetrica' for architecture arm64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Analyzing workspace
    note: Constructing build description
    note: Build preparation complete

```

**Comment by formatCvt** [comment]
Thanks for reporting! I will check this week

**Comment by formatCvt** [comment]
Sorry for long delay, try version 0.7.0+1 please

**Comment by glin94** [comment]
хорошо

**Comment by otopba** [comment]
@glin94 Do you have a recurring error or is it no longer there after a version update?

**Comment by JRakhimov** [comment]
@formatCvt @otopba There is new error when running in ios simulator:

```
[!] CocoaPods could not find compatible versions for pod "YandexMobileMetrica/Dynamic":
  In Podfile:
    appmetrica_sdk (from `.symlinks/plugins/appmetrica_sdk/ios`) was resolved to 0.0.1, which depends on
      YandexMobileMetrica/Dynamic (= 4.0.0)

None of your spec sources contain a spec satisfying the dependency: `YandexMobileMetrica/Dynamic (= 4.0.0)`.

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.
 ```
 
 and 
 
 ```
 [!] CocoaPods could not find compatible versions for pod "YandexMobileMetrica/Dynamic":
  In snapshot (Podfile.lock):
    YandexMobileMetrica/Dynamic (= 3.14.0)

  In Podfile:
    appmetrica_sdk (from `.symlinks/plugins/appmetrica_sdk/ios`) was resolved to 0.0.1, which depends on
      YandexMobileMetrica/Dynamic (= 4.0.0)

Specs satisfying the `YandexMobileMetrica/Dynamic (= 3.14.0), YandexMobileMetrica/Dynamic (= 4.0.0)` dependency were found, but they required a higher minimum deployment target.
```
